dags/inference/configs/maxtext_inference_microbenchmark_gce_config.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utilities to construct configs for maxtext inference microbenchmarks DAG."""
+
+import datetime
+import json
+from typing import Dict
+from xlml.apis import gcp_config, metric_config, task, test_config
+from dags import test_owner
+from dags.multipod.configs import common
+from dags.vm_resource import TpuVersion, Project, RuntimeVersion
+
+PROJECT_NAME = Project.CLOUD_ML_AUTO_SOLUTIONS.value
+RUNTIME_IMAGE = RuntimeVersion.TPU_UBUNTU2204_BASE.value
+GCS_SUBFOLDER_PREFIX = test_owner.Team.INFERENCE.value
```

Comment:
This is great! Do we have a recommendation on the test_owner? i.e. if others need to run it, they are the test owner right?

Reply:
The `test_owner` variable is a bit misleading. It refers to a dataclass, with the attribute as just the string "inference". So in GCS, when we upload the jsonlines file, it will be in `gs://..../inference/maxtext/...`.

If others want to run, they would add/change `task_owner` below.